rxgroups/src/main/java/com/airbnb/rxgroups/ProcessorUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.airbnb.rxgroups;
+
+
+import java.util.List;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.type.DeclaredType;
+import javax.lang.model.type.TypeKind;
+import javax.lang.model.type.TypeMirror;
+
+class ProcessorUtils {
+
+  static boolean isResubscribingObserver(Element observerFieldElement) {
```

Comment:
I tested it, and it seems to work okay. But I wonder why Butterknife choose to do this way..